src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -134,33 +157,36 @@ private boolean isExceptionRecordCreated(UUID randomPoBox) {
             )
         );
 
-        exceptionRecordCase = caseDetailsList.isEmpty() ? null : caseDetailsList.get(0);
-        return exceptionRecordCase != null;
+        exceptionRecord = caseDetailsList.isEmpty() ? null : caseDetailsList.get(0);
+        return exceptionRecord != null;
     }
 
-    private void verifyExistingCaseIsUpdatedWithExceptionRecordCaseDetails() {
+    private Boolean isExceptionRecordAttachedToTheCase(int expectedScannedDocsSize) {
```

Comment:
I dont think this is specific enough for a test assertion.
We should be asserting on the document number and the dm reference at the very least.

These should be passed in via a varargs parameter.

Reply:
@KevinJCross I've updated the test now. Could you please take another look?
assertThat(updatedScannedDocuments).containsAll(exceptionRecordDocuments);

the above assertion verifies all the fields in the ScannedDocument fields in the ScannedDcoument-> equals() method